libraries/state_history/include/sysio/state_history/serialization.hpp

```diff
@@ -386,7 +386,7 @@ datastream<ST>& operator<<(datastream<ST>& ds, const history_serial_wrapper_stat
    fc::raw::pack(ds, fc::unsigned_int(0));
    fc::raw::pack(ds, as_type<uint64_t>(obj.obj.sender.to_uint64_t()));
    fc::raw::pack(ds, as_type<__uint128_t>(obj.obj.sender_id));
-   fc::raw::pack(ds, as_type<uint64_t>(obj.obj.payer.to_uint64_t()));
+   // fc::raw::pack(ds, as_type<uint64_t>(obj.obj.payer.to_uint64_t()));
```

Comment:
Based on the decision from the other comment, remove commented out code or else this whole method